www/components/AgendaIcon.js

```diff
@@ -0,0 +1,17 @@
+import CalendarTodayRoundedIcon from '@material-ui/icons/CalendarTodayRounded';
+import AccessAlarmRoundedIcon from '@material-ui/icons/AccessAlarmRounded';
+import styles from "../styles/AgendaIcon.module.css"
+function AgendaIcon() {
+    return (
+        <div className={styles.contain}>
+            <CalendarTodayRoundedIcon style={{fontSize: 35,
+            color: "var(--agendaSecondary)"}}/>
+            <div className={styles.check}>
+                <AccessAlarmRoundedIcon style={{fontSize: 17, 
```

Comment:
Generally inline styles are a bad practice but since this is inside of an icon meaning that the styles will only be used once it is probably fine. This issue is when people go looking for styles and can't find them in the CSS classes.

Reply:
Ya we did all icons styles that way in the buttons too